apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/oms/OmstillingsstoenadInnvilgelse.kt

```diff
@@ -53,9 +53,10 @@ data class OmstillingsstoenadInnvilgelse(
                         restanse = it.restanse,
                         utbetaltBeloep = it.utbetaltBeloep,
                         trygdetid = it.trygdetid,
-                        beregningsMetodeFraGrunnlag = it.beregningsMetodeFraGrunnlag,
                         beregningsMetodeAnvendt = it.beregningsMetodeAnvendt,
+                        beregningsMetodeFraGrunnlag = it.beregningsMetodeFraGrunnlag,
                         sanksjon = it.sanksjon != null,
+                        institusjon = it.institusjon != null,
```

Comment:
todo: Må kanskje se på innholdet for å avgjøre?

Reply:
@sebassonav : Noen innspill på om dette er nok, eller bør man sjekke på om `reduksjon` er mer enn 0% f. eks?

Reply:
Dette skal være nok hvis du lurer på om sb har lagt inn opphold.